bundle/permissions/mutator.go

```diff
@@ -26,11 +31,11 @@ var levelsMap = map[string](map[string]string){
 		CAN_VIEW:   "CAN_VIEW",
 		CAN_RUN:    "CAN_RUN",
 	},
-	"mlflow_experiments": {
+	"experiments": {
 		CAN_MANAGE: "CAN_MANAGE",
 		CAN_VIEW:   "CAN_READ",
 	},
-	"mlflow_models": {
+	"models": {
```

Comment:
Given that the lookup for this map used the previous keys, this change is a nop for experiments and models?

Reply:
The rename is to make sure this key matches the one defined in bundle configuration for this resource type